app/views/catalog/_home.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class='jumbotron'>
+  <div class='row'>
+    <h1><%= t('arclight.views.home.heading') %></h1>
+  </div>
+  <div class='col-md-6 col-md-offset-3'>
```

Comment:
I wonder if we should make this a bit larger on desktop (just using the bootstrap classes).  Perhaps that got more complicated than I'm making it sound, so totally understand if we want to punt and work on the proper sizing later.